test/e2e/utils/chaise.page.js

```diff
@@ -890,6 +890,18 @@ var recordsetPage = function() {
         return element.all(by.css(".table-column-displayname > span"));
     };
 
+    this.getColumnSortButton = function(index){
+        return element(by.id("divRecordSet")).all(by.css('table > thead > tr > th')).get(index).element(by.css('.glyphicon-sort'));
```

Comment:
This morning I just merged some new changes to the table directive that adds classes to each `table > thead > tr > th`.  If you pull those changes into your branch, you can simplify all three of these selectors to be:
```
this.getColumnSortButton = function(rawColumnName){
  return element(by.css('.' + rawColumnName)).element(by.css('.glyphicon-sort-by-attributes-alt'));
}
```
or simplify it more into a single selector:
```
element(by.css('.' + rawColumnName + ' .glyphicon-sort-by-attributes-alt'));
```

Reply:
I think `rawColumnName` is the same as the expected data property you added called `nameInUrl`.